www/include/monitoring/status/ServicesHostGroups/serviceGridByHGJS.php

```diff
@@ -155,8 +155,8 @@ function initM(_time_reload, _sid, _o) {
         if (document.getElementById("select2-hg_search-container") &&
             document.getElementById("select2-hg_search-container").title
         ) {
-            this.hg_search = document.getElementById("select2-hg_search-container").title;
-            viewDebugInfo('search: ' + document.getElementById("select2-hg_search-container").title);
+            this.hg_search = jQuery("#select2-hg_search-container .select2-content").attr("title");
```

Comment:
search is the title of select2 ?

Reply:
It seems it's easier to get the title of selected option instead of content of the span